tests/bettertransformer/test_bettertransformer_encoder.py

```diff
@@ -87,6 +88,10 @@ def _loop_all_classes(self):
             elif layer_class == "TransformerBlock":
                 # Hardcode it for distilbert - see https://github.com/huggingface/transformers/pull/19966
                 class_name = "DistilBert"
+
+            elif layer_class == "TapasLayer":
+                class_name = "Tapas"
+
```

Comment:
```suggestion
            elif layer_class == "TapasLayer":
                class_name = "Tapas"
```
No need for the new lines.